docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -577,7 +577,92 @@ You should be redirected again to the login page at Keycloak. However, now you a
 In both cases, if the user is successfully authenticated, the landing page will show the user's name and e-mail. Even though
 user `alice` exists in both tenants, for the application they are distinct users belonging to different realms/tenants.
 
-== Resolving Tenant Identifiers with Annotations
+[[static-tenant-resolution]]
+== Static Tenant configuration resolution
```

Comment:
Sentence case as per Quarkus title style:
```suggestion
== Static tenant configuration resolution
```

If this heading is a required step in this how-to guide to complete the end-to-end solution, use a more [user-focussed and goal-oriented heading](https://quarkus.io/version/main/guides/doc-reference#titles-headings), for example : 

  ```suggestion
== Configure the resolution of static tenants
```

or 

  ```suggestion
== Configure static tenant resolution
```